command/intention/create/create_test.go

```diff
@@ -133,6 +138,35 @@ func TestCommand_meta(t *testing.T) {
 	require.Equal(map[string]string{"hello": "world"}, ixns[0].Meta)
 }
 
+// Test that creating an intention with -source-type != consul fails
+// since not Consul Enterprise.
```

Comment:
How will this test not fail in Consul enterprise? It may need to be in a separate file with a build tag?